### DISCUSSION TITLE
Dotnet run?

### DISCUSSION THREAD

**Discussion by DarkShadoYT** [drizzle-mizzle/CharacterAI-Discord-Bot#85] [Q&A] [answered]
Is there a way to run this without the dotnet run because both files don't run and my school laptop doesn't let me get the . NetSDK

**Comment by drizzle-mizzle** [comment, marked as answer]
Just download a precompiled binaries...

**Reply by DarkShadoYT** [reply]
What in the world is that 😓😅 (sorry)

**Reply by drizzle-mizzle** [reply]
![20230502_115906.png](https://user-images.githubusercontent.com/55811932/235600420-1656c1e7-de3e-48b5-a838-6e4f9fed758a.png)

**Reply by DarkShadoYT** [reply]
Oh 🤦🏾 I'm slow
